src/annotator/sidebar.js

```diff
@@ -19,8 +19,6 @@ import { ToolbarController } from './toolbar';
 const MIN_RESIZE = 280;
 
 const defaultConfig = {
-  Document: {},
-  TextSelection: {},
```

Comment:
There is no `TextSelection` plugin any more and the `Document` config is already supplied by the base class.